### PR TITLE
Enable CI on Nano Server

### PR DESCRIPTION
**Pull request by MichaelSimons**
cc @naamunds

### CODE REVIEW

**Comment by MichaelSimons** [comment]
test ci please

**Comment by MichaelSimons** [comment]
test ci please

File: netci.groovy [hunk 1]
```diff
@@ -3,26 +3,32 @@ import jobs.generation.Utilities
...
+    def newJob = job(newJobName) {
+        steps {
+            if (hostOS == 'Windows_2016')
+            {
+                batchFile("powershell -NoProfile -Command .\\build-and-test.ps1")
+            }
+            else if (hostOS == 'Windows Nano 2016')
+            {
+                batchFile("powershell -NoProfile -Command .\\build-and-test.ps1 -OS nanoserver")
+                machineLabel = ''
```
Comment (naamunds) [inline netci.groovy]: It feels a bit strange to me to set `machineLabel` inside the definition of the job's steps.  What do you think about just initializing it to its correct value by using the ternary operator on line 11?

File: netci.groovy [hunk 2]
```diff
@@ -3,26 +3,32 @@ import jobs.generation.Utilities
...
-			{
-				shell("./build-and-test.sh")
-			}
-		}
-	}
+    def newJob = job(newJobName) {
+        steps {
+            if (hostOS == 'Windows_2016')
+            {
+                batchFile("powershell -NoProfile -Command .\\build-and-test.ps1")
```
Comment (naamunds) [inline netci.groovy]: Would it preferable to use the `OS` parameter here also, rather than relying on its default value?

File: netci.groovy [hunk 3]
```diff
@@ -3,26 +3,32 @@ import jobs.generation.Utilities
...
-			else
-			{
-				shell("./build-and-test.sh")
-			}
-		}
-	}
+    def newJob = job(newJobName) {
+        steps {
+            if (hostOS == 'Windows_2016')
+            {
```
Comment (naamunds) [inline netci.groovy]: Consider putting this brace on the previous line for formatting consistency.  Same with the ones for "else if" and "else".

Comment (MichaelSimons) [inline netci.groovy]: Fixed

---
In reply to: [79723933](https://github.com/dotnet/dotnet-docker-nightly/pull/51#discussion_r79723933) [](ancestors = 79723933)

File: netci.groovy (hunk 1 above)
Comment (MichaelSimons) [inline netci.groovy]: Agreed - fixed

---
In reply to: [79722462](https://github.com/dotnet/dotnet-docker-nightly/pull/51#discussion_r79722462) [](ancestors = 79722462)

File: netci.groovy (hunk 2 above)
Comment (MichaelSimons) [inline netci.groovy]: Not a big deal but changed as suggested.

---
In reply to: [79722600](https://github.com/dotnet/dotnet-docker-nightly/pull/51#discussion_r79722600) [](ancestors = 79722600)

**Comment by MichaelSimons** [comment]
test ci please

**Pull request merged**